docs/tutorials/python/ndarray.md

```diff
@@ -30,15 +30,16 @@ They can be initialized in various ways:
     >>> b[:] = 2 # set all elements of b to 2
 ```
 
-We can copy the value from one `NDArray` to another, even if they are located on different devices:
+We can copy the values from one `NDArray` to another, even if they are located
+on different devices:
 
 ```python
     >>> a = mx.nd.ones((2, 3))
     >>> b = mx.nd.zeros((2, 3), mx.gpu())
     >>> a.copyto(b) # copy data from cpu to gpu
 ```
 
-We can also convert `NDArray` to `numpy.ndarray`:
+We can also convert a `NDArray` to a `numpy.ndarray`:
```

Comment:
"a" -> "an" because `NDArray` starts with a *vowel sound* (even though n is a consonant)